lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -114,18 +121,26 @@ def ext_exists(cursor, ext):
     return cursor.rowcount == 1
 
 
-def ext_delete(cursor, ext):
+def ext_delete(cursor, ext, cascade):
     if ext_exists(cursor, ext):
-        query = "DROP EXTENSION \"%s\"" % ext
+        if cascade:
```

Comment:
Would it be better to change the logic here to be:
```python
        query = "DROP EXTENSION \"%s\"" % ext
        if cascade:
            query += " CASCADE"
```

Reply:
Thank you. It's fixed now